web/pdf_thumbnail_view.js

```diff
@@ -378,7 +373,11 @@ class PDFThumbnailView {
   }
 
   #reduceImage(img) {
-    const { ctx, canvas } = this.#getPageDrawContext(1, true);
+    const { canvas } = this.#getPageDrawContext(1);
+    const ctx = canvas.getContext("2d", {
+      alpha: false,
+      willReadFrequently: false,
```

Comment:
This no longer depends on `enableHWA`; is that expected?

Reply:
Good point! In that case, since there is another call to `#getPageDrawContext` in this file that didn't pass the parameter and thus defaulted to `this.enableHWA`, can we remove all `enableHWA` usage from this file now that that member variable doesn't appear to be used anywhere (I think that's partly where my confusion came from).

Reply:
However, it looks like `enableHWA` in this file is now unused, so can we remove all references to it to simplify things further? AFAICT we currently still set e.g. `this.enableHWA` even though nothing seems to read from it anymore.

Reply:
I went around checking the flow of that info and realized it was needed atleast until we want to make the `getContext` call in `InternalRenderTask` due to the clarification in https://github.com/mozilla/pdf.js/pull/20016#discussion_r2181079795.